docs/loading/Loading_data_template.md

```diff
@@ -0,0 +1,348 @@
+# Load data from \<SOURCE\> TEMPLATE
+
+## Template instructions
+
+### A note about style
+
+Technical documentation typically has links to other documents all over the place. When you look at this document you may notice that there are few links out from the page, and that almost all of the links are at the bottom of the doc in the **more information** section. Not every keyword needs to be linked out to another page, please assume that the reader knows what `CREATE TABLE` means and that is they do not know they can click in the search bar and find out. It is fine to pop a note in the docs to tell the reader that there are other options and the details are described in the **more information** section; this let's the people who need the information know that they can read it ***later***, after they accomplish the task at hand.
+
+### The template
+
+This template is based on the process to load data from Amazon S3, some
+parts of it will not be applicable to loading from other sources. Please concentrate on the flow of this template and do not worry about including every section; the flow is meant to be:
+
+#### Introduction
+
+Introductory text that let's the reader know what the end result will be if they follow this guide. In the case of the S3 doc, the end result is "Getting data loaded from S3 in either an asynchronous manner, or a synchronous manner.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/acf3177b-567d-4ccf-9d6b-f9363d42f7c0)
The closing quotation mark is missing I think.